nixos/modules/programs/nix-monitored.nix

```
@@ -0,0 +1,40 @@
+{ pkgs, lib, config, ... }:
+
+let
+  cfg = config.nix.monitored;
+in
+{
+  meta.maintainers = [ lib.maintainers.ners ];
+
+  options.nix.monitored = with lib; {
```

```suggestion
  options.nix.monitored = {
```

Use `lib` where necessary.